md-dao/src/main/java/com/wipro/ats/bdre/md/dao/InstanceExecDAO.java

```diff
@@ -160,4 +162,43 @@ public void delete(Long id) {
             session.close();
         }
     }
+
+
+   public List<SLAMonitoringBean> slaMonitoringData(List<Process> subProcessList)
+    {
+        Session session = sessionFactory.openSession();
+        List<SLAMonitoringBean> slaMonitoringBeanList=new ArrayList<>();
+        try {
+            session.beginTransaction();
+            int counter=subProcessList.size();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "counter" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1481)